src/vendor/GLTFLoader.js

```diff
@@ -1813,6 +1832,8 @@ THREE.GLTFLoader = ( function () {
 
 		}
 
+		var farsparkURL = await toFarsparkURL(resolveURL(sourceURI, options.path));
+
 		return Promise.resolve( sourceURI ).then( function ( sourceURI ) {
```

Comment:
I want to use the `sourceURI` here because it ends in (for example) `.jpg`, which may match a regex in `THREE.Loader.Handlers.get` to give back the loader for that filetype. Farspark won't preserve the file extension so that would change the way this code works. (I don't know if `THREE.Loader.Handlers.get( sourceURI )` every succeeds in finding the loader, but I was just trying not to change the logic at all besides swapping the loaded url at the last moment.